demisto_sdk/commands/common/hook_validations/base_validator.py

```diff
@@ -421,3 +422,46 @@ def validate_xsiam_content_item_title(file_path):
             if file_name != schema_expected_name:
                 return False
         return True
+
+    @error_codes("BA125")
+    def validate_no_disallowed_terms_in_customer_facing_docs(
+        self, file_content: str, file_path: str
+    ) -> bool:
+        """
+        Validate that customer facing docs and fields don't contain any internal terms that aren't clear for customers.
+
+        Args:
+            file_content (str): The content of the file to check.
+            file_path (str): The path of the file the content belongs to.
+
+        Returns:
+            bool: True if no such terms were found, False otherwise.
+        """
+        disallowed_terms = (
+            [  # These terms are checked regardless for case (case-insensitive)
+                "test-module",
+                "test module",
+                "long-running-execution",
+            ]
+        )
```

Comment:
It would be better practice to put this list in a different file/ const then the code, it might grow significally with time.

Reply:
That's a good point, but I think it'll be more fitting to just move it once and if it gets large, instead of moving it now to a file with just 3 terms, which will make it slightly less readable (having to refer to another file to see what the disallowed terms are).

The ticket I opened has just 3 additional potential terms that might be added.

Reply:
I would use const at least, but lets close it and handle that in the future.